drupal-developer-portal/README.md

```diff
@@ -0,0 +1,140 @@
+# Drupal Developer Portal
+
+This sample lets you create a Drupal developer portal for your Apigee API product
+
+## About Drupal developer portals
+
+Apigee's Drupal developer portal enables users to quickly and easily stand up a highly customizable developer portal for their APIs. Unlike the Drupal developer portal, the Drupal portal isn't managed by Apigee. So we will use the Google Cloud Platform (GCP) Marketplace solution to deploy the portal's infrastructure. To learn more about Apigee Drupal portals, see the [Google documentation](https://cloud.google.com/apigee/docs/api-platform/publish/drupal/open-source-drupal).
+
+## Implementation on Apigee 
+
+The Apigee proxy sample uses only a few policies:
+1. An API Key policy to verify incoming request API Key credentials
+2. A CORS policy to allow requests from the developer portal webpage
+
+## Prerequisites
+1. [Provision Apigee X](https://cloud.google.com/apigee/docs/api-platform/get-started/provisioning-intro)
+2. Configure [external access](https://cloud.google.com/apigee/docs/api-platform/get-started/configure-routing#external-access) for API traffic to your Apigee X instance
+3. Have access to deploy proxies, create products, and provision a portal in Apigee
+4. Make sure the following tools are available in your terminal's $PATH (Cloud Shell has these preconfigured)
+    * [gcloud SDK](https://cloud.google.com/sdk/docs/install)
+    * unzip
+    * curl
+    * jq
+    * npm
+5. You have deployed the Marketplace Drupal developer portal as described in the [documentation](https://cloud.google.com/apigee/docs/api-platform/publish/drupal/get-started-cloud-marketplace). Be sure to enable HTTPS under Networking during portal configuration. No further customizations are needed.
```

Comment:
I think we might want to add this as a step in the tutorial itself.  Simply listing this as a prerequisite here might be confusing to a new user.